Library/Homebrew/env_config.rb

```diff
@@ -289,6 +289,10 @@ module EnvConfig
                      "outdated.",
         boolean:     true,
       },
+      HOMEBREW_NO_UPDATE_REPORT_ONLY_INSTALLED:  {
+        description: "If set, `brew update` lists updates to installed software.",
```

Comment:
```suggestion
        description: "If set, `brew update` lists updates to all software.",
```

Unless I've misunderstood this, `brew update` will list updates to installed software only by default and setting this environment variable makes `brew update` list updates to _all_ software (not just installed). The description seems to describe the new default behavior instead of the behavior with this environment variable set.